xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/LivedataPagination.vue

```diff
@@ -57,6 +57,7 @@
     >
       {{ $t('livedata.pagination.pageSize') }}
       <select
+        :title="$t('livedata.pagination.selectPageSize')"
```

Comment:
Don't you also need to add this key to https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/Logic.js#L199 ?

Reply:
I didn't spot this issue since this didn't fail the build, and when I test on my local instance, I always just append translations to an existing page.